requirements/edx/base.txt

```diff
@@ -110,7 +111,8 @@ edx-enterprise==1.9.12
 edx-i18n-tools==0.4.8
 edx-milestones==0.2.3
 edx-oauth2-provider==1.3.0
-edx-opaque-keys[django]==1.0.1
+# edx-opaque-keys[django]==1.0.1 temporarily replaced by the following until https://github.com/edx/opaque-keys/pull/108 is merged:
+git+https://github.com/open-craft/opaque-keys.git@4f784d98c1f0eece0f5c293fabc66e43aa28d166#egg=edx-opaque-keys==1.1.0
```

Comment:
Set these back now that we've got the keys related PRs merged?

Reply:
Yeah once it's merged I'll rebase and this part of the diff will all go awawy